spec/controllers/admin/suspend_users_controller_spec.rb

```diff
@@ -61,6 +62,19 @@
       end
     end
 
+    context "when external IDs are provided" do
+      let(:external_ids_to_suspend) { users_to_suspend.map { |user| user.external_id } }
+      let(:specified_ids) { external_ids_to_suspend.join(", ") }
+
+      it "enqueues a job to suspend the specified users" do
+        put :update, params: { suspend_users: { identifiers: specified_ids, reason: } }
```

Comment:
Nit: you can factor this request in a `before` block under the `describe` level so you don't have to invoke if for all three contexts